cli/src/cli_util.rs

```diff
@@ -252,6 +253,17 @@ impl CommandHelper {
         Ok(template_builder::parse(language, template_text, &aliases)?)
     }
 
+    pub fn parse_operation_template(
+        &self,
+        ui: &Ui,
+        template_text: &str,
+        root_op_id: &OperationId,
+        current_op_id: Option<&OperationId>,
+    ) -> Result<Box<dyn Template<Operation>>, CommandError> {
+        let language = OperationTemplateLanguage::new(root_op_id, current_op_id);
+        self.parse_template(ui, &language, template_text)
+    }
```

Comment:
nit: I would inline it in commands/operation.rs (because there are no other callers), but I understand that it's also a bit weird that `CommandHelper` had a getter of `&OperationTemplateLanguageExtension`.